### PR TITLE
fix(service): use correct endpoint

### PR DESCRIPTION
**Pull request by samouss**

### CODE REVIEW

File: 08-service/README.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ $ minikube ip
 192.168.99.100
 ```
 
-Open a browser on `http://192.168.99.100/info`
+Open a browser on `http://192.168.99.100/health`
```
Comment (samouss) [inline 08-service/README.md]: The endpoint does not exist on 0.4.0 only on [0.5.0+](https://github.com/mhausenblas/simpleservice#info-050).

**Comment by ElPicador** [comment]
@samouss thanks, but I can't merge 😢

**Comment by samouss** [comment]
Me neither... BTW, great content! Thanks a lot!

**Comment by seddarj** [comment]
Hello, thanks for the PR. I'm going to close in favour of #60.

**Pull request closed without merging**